recipes/apc.rb

```diff
@@ -25,18 +25,19 @@
   only_if { node['osl-php']['use_ius'] }
 end
 
-include_recipe 'build-essential'
-
 %w(httpd-devel pcre pcre-devel).each do |pkg|
   package pkg do
     action :install
   end
 end
 
-php_pear 'APC' do
+build_essential 'apc'
+
+php_pear 'apc' do
   action :install
+  channel 'pecl'
```

Comment:
Is there a reason why we need to set the channel here? If so, can you please add a comment?